### ISSUE TITLE
Report packages as not "currently no response from maintainer" / "maybe deprecated"

### ISSUE THREAD

**Issue by JHGitty** [composer/packagist#600]
Currently the maintainer of a package can set a packagist as deprecated.

But there are sometimes the issue that the maintainer of a *widely used package* is not responding and the bundle itself is deprecated.

I think there should be an option to report an package as "maybe deprecated" or "the maintainer does not respond". There should be a guideline how this issue will be handled.

Ideas:
1. Only packages which are not updated for X months should be reportable.
2. If a package is reported the maintainer will get an notification.

What do you think about it? What is the current situation to handle this issue?

**Comment by Seldaek** [comment]
Not sure if we really have to do this, it is also up to people to check a bit the state of a package before using it IMO. I'm also not sure how to best implement it from a UX perspective.. reporting is one thing, but it'd be nice if people could suggest a replacement then, but if people don't agree on a replacement it's a mess.

**Comment by JHGitty** [comment]
https://github.com/misd-service-development/phone-number-bundle/issues/63#issuecomment-151867925

**Comment by Seldaek** [comment]
Yeah I saw that but still not sure if it isn't best to handle things by talking to people rather than hitting a "report" button. That doesn't resolve anything in the end to mark the package abandoned. What we need is find a new maintainer etc.

**Comment by Rvanlaak** [comment]
Think the problem mainly lies in the (Github) repository access, there wouldn't be any problem if the maintainer would take a minute to add someone :+1: So the next question is or Packagist should be the one to take more responsibility for abandoned packages.

**Comment by jakoch** [comment]
> But there are sometimes the issue that the maintainer of a widely used package is not responding and the bundle itself is deprecated.

One more issue to think about: the endpoint of a package might be dead and gone.
For example https://packagist.org/packages/lastcraft/simpletest → https://github.com/lox/simpletest
A report button to notify the package maintainer could help.
